docs/user_guides/fs/storage_connector/creation/redshift.md

```diff
@@ -59,38 +64,40 @@ Enter the details for your Redshift connector. Start by giving it a **name** and
 
     Your administrator can change the default session duration for AWS storage connectors, by first [increasing the max session duration of the IAM Role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use.html#id_roles_use_view-role-max-session) that you are assuming. And then changing the `fs_storage_connector_session_duration` [configuration property](../../../../admin/variables.md) to the appropriate value in seconds.
 
-### Step 3: Upload the Redshift database driver
 ### Step 3: Upload the Redshift database driver (optional)
 
 The `redshift-jdbc42` JDBC driver is included by default in the Hopsworks distribution. 
 If you wish to use a different driver, you need to upload it on Hopsworks and add it as a dependency of Jobs and Jupyter Notebooks that need it. First, you need to [download the library](https://docs.aws.amazon.com/redshift/latest/mgmt/jdbc20-download-driver.html). Select the driver version without the AWS SDK.
 
-#### Upload the driver to Hopsworks
-You then upload the driver files to the “Resources” directory in your project, see the screenshot below.
-
-1. Open the File Browser.
-2. Move into the "Resources" directory.
-3. Upload the driver jar file.
-
-<figure markdown>
-  ![Redshift Driver Upload](../../../../assets/images/guides/fs/storage_connector/driver_upload.png)
-  <figcaption>Redshift Driver Upload in the Filesystem Browser</figcaption>
-</figure>
-
 #### Add the driver to Jupyter Notebooks and Spark jobs
 
 You can now add the driver file to the default job and Jupyter configuration. This way, all jobs and Jupyter instances in the project will have the driver attached so that Spark can access it.
 
 1. Go into the Project's settings.
 2. Select "Compute configuration".
 3. Select "Spark".
-4. Select and Upload the jar file in the "additional jars" section of the configuration.
+4. Under "Additional Jars" choose to "Upload new file" to upload the driver jar file.
 
 <figure markdown>
-  ![Redshift Driver Job and Jupyter Configuration](../../../../assets/images/guides/fs/storage_connector/driver_upload.png)
+  ![Redshift Driver Job and Jupyter Configuration](../../../../assets/images/guides/fs/storage_connector/jupyter_config.png)
   <figcaption>Attaching the Redshift Driver to all Jobs and Jupyter Instances of the Project</figcaption>
 </figure>
 
+Alternatively, you can choose "From Project" option to choose an already existing jar file in the Project. For this, 
+you will first have to upload a jar file in Project using the File Browser if not already, and then choose the 
+desired file using the "From Project" option. To upload the jar file in Project through File Browser, see below example:
```

Comment:
To upload the jar file to the Project through the File Browser, see the example below: